An error occurred when trying to build with Maven 3.
Please tell me how to resolve.
[ERROR] Failed to execute goal on project mcMMO-WorldGuard: Could not resolve dependencies for project org.mcmmo:mcMMO-WorldGuard:jar:1.0.4: Failure to find com.gmail.nossr50.mcMMO:mcMMO:jar:1.5.08.204 in https://hub.spigotmc.org/nexus/content/repositories/snapshots/ was cached in the local repository, resolution will not be reattempted until the update interval of spigot-repo has elapsed or updates are forced -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

--- a/Changelog.txt
+++ b/Changelog.txt
@@ -7,6 +7,9 @@ Key:
   ! Change
   - Removal
 
+Version 1.0.5
+ ! Changed mcMMO to 1.5.09-SNAPSHOT
+
 Version 1.0.4
  ! Built using Bukkit 1.12.1-R0.1-SNAPSHOT
  ! Built using mcMMO 1.5.08 build 204

--- a/pom.xml
+++ b/pom.xml
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.mcmmo</groupId>
     <artifactId>mcMMO-WorldGuard</artifactId>
-    <version>1.0.4</version>
+    <version>1.0.5</version>
     <name>mcMMO-WorldGuard</name>
     <url>https://github.com/mcMMO-Dev/mcMMO-WorldGuard</url>
     <issueManagement>
@@ -46,7 +46,7 @@
                             <repositoryLayout>default</repositoryLayout>
                             <groupId>com.gmail.nossr50.mcMMO</groupId>
                             <artifactId>mcMMO</artifactId>
-                            <version>1.5.08.204</version>
+                            <version>1.5.09-SNAPSHOT</version>
                             <packaging>jar</packaging>
                             <generatePom>true</generatePom>
                         </configuration>

